packages/language-server/src/server.ts

```diff
@@ -25,15 +27,33 @@ import {
   hasSettingsForDocument,
   getSettingsForDocument,
 } from "@ui5-language-assistant/settings";
+import { ServerInitializationOptions } from "../api";
+import { getXMLViewDiagnostics } from "./xml-view-diagnostics";
+import { getHoverResponse } from "./hover";
+
+let workspaceFolderUri: string | null;
+let workspaceFolderPath: string | null;
+type absolutePath = string;
+type manifestsData = Record<absolutePath, { isFlexEnabled: boolean }>;
+const manifestData: manifestsData = Object.create(null);
```

Comment:
`manifestsData` name is used for both type and const.
can we differentiate those? e.g:
- type **M**anifestData (upper case M)